.github/workflows/vagrant-box-atom.yml

```diff
@@ -0,0 +1,36 @@
+name: AtoM Vagrant box
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version
+        required: true
+      description:
+        description: Description
+        required: true
+jobs: 
+  vagrant-box-atom:
+    name: Build and upload
+    runs-on: macos-10.15
```

Comment:
MacOS required for nested virtualization:

https://github.com/actions/virtual-environments/issues/183
https://github.com/actions/virtual-environments/issues/433

It also comes with all the required dependencies (Virtualbox, Vagrant, Packer, Bundler).